### PR TITLE
add permission denied test

### PR DESCRIPTION
**Pull request by ouwenkg**
### Description of the Change

1. Add permission denied test
2. Put common test tools together into `txtool_utils.py`, and make py-tests more clear. 
3. Seperate features relevant test from original `cita_charge_mode.py`, only transfer cases remained
now.

### Applicable Issues

#650

### CODE REVIEW

File: tests/integrate_test/cita_charge_mode.sh [hunk 1]
```diff
@@ -54,6 +47,7 @@ main() {
     echo -n "1) generate config  ...  "
     create_config \
         --contract_arguments "SysConfig.checkFeeBackPlatform=true" \
+        --contract_arguments "SysConfig.checkCreateContractPermission=true" \
```
Comment (kaikai1024) [inline tests/integrate_test/cita_charge_mode.sh]: No need.
and `--contract_arguments "SysConfig.checkFeeBackPlatform=true"`

File: tests/integrate_test/cita_features_test.sh [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    . ${SOURCE_DIR}/tests/integrate_test/util.sh
+    cd "${BINARY_DIR}"
+    echo "DONE"
+
+    echo -n "1) generate config  ...  "
+    create_config \
+        --contract_arguments "SysConfig.checkFeeBackPlatform=true" \
+        --contract_arguments "SysConfig.checkCreateContractPermission=true" \
+        --contract_arguments "SysConfig.economicalModel=1" \
+        --contract_arguments "VersionManager.version=0" \
```
Comment (kaikai1024) [inline tests/integrate_test/cita_features_test.sh]: What about version 1 and 2?
Why do not use version 2 directly?

File: tests/integrate_test/cita_charge_mode.sh (hunk 1 above)
Comment (ouwenkg) [reply tests/integrate_test/cita_charge_mode.sh]: permission setting is needless, checkFeeBackPlatform is used while checking transferd balance.

File: tests/integrate_test/cita_features_test.sh (hunk 2 above)
Comment (ouwenkg) [reply tests/integrate_test/cita_features_test.sh]: I just copy from `cita_charge_mode`, version=2 is ok.

File: tests/integrate_test/cita_charge_mode.sh (hunk 1 above)
Comment (kaikai1024) [reply tests/integrate_test/cita_charge_mode.sh]: Is not checkFeeBackFlatform used for fee_back test?

File: tests/integrate_test/cita_features_test.sh (hunk 2 above)
Comment (kaikai1024) [reply tests/integrate_test/cita_features_test.sh]: ok.
`cita_charge_mode` testes both version and version 1. Your copy seems miss version 1.

File: tests/integrate_test/cita_charge_mode.sh (hunk 1 above)
Comment (ouwenkg) [reply tests/integrate_test/cita_charge_mode.sh]: Setting `SysConfig.checkFeeBackPlatform=true`, the transaction fee goes to chain owner,
when remove this config, tx fees goes to miner which causes original transfer test cases fail.

I do not want to change orginal test logic, so remain checkFeeBackPlatform=true.

Comment (kaikai1024) [reply tests/integrate_test/cita_charge_mode.sh]: I see.

File: tests/integrate_test/cita_features_test.sh (hunk 2 above)
Comment (ouwenkg) [reply tests/integrate_test/cita_features_test.sh]: I remove version config at start, just use latest version 2

**Pull request merged**